php_phongo.c

```diff
@@ -1880,11 +1878,17 @@ PHP_MINIT_FUNCTION(mongodb)
 {
 	(void)type; /* We don't care if we are loaded via dl() or extension= */
 
+	char *php_version_string = malloc(4 + sizeof(MONGODB_VERSION_S) + 1);
```

Comment:
_append_and_truncate may free then.

With emalloc, I got a segfault *or* after moving the call to after the libson vtable init I got a segfault. It *does* free it in some way.

https://github.com/mongodb/mongo-c-driver/blob/bc116fe08857837980a64b36bce8beac7d9c4874/src/mongoc/mongoc-handshake.c#L459